spec/controllers/application_controller/advanced_search_spec.rb

```diff
@@ -1,18 +1,20 @@
-describe ProviderForemanController, "::AdvancedSearch" do
-  before do
-    stub_user(:features => :all)
-    controller.instance_variable_set(:@sb, {})
-  end
+describe EmsConfigurationController, "::AdvancedSearch" do
```

Comment:
Not for this PR, but it's weird to me to have a `EmsConfigurationController, "::AdvancedSearch"` in a file named `application_controller/advanced_search_spec.rb ` as the latter is generic and the former is specific.